pymongo/encryption.py

```diff
@@ -33,11 +33,11 @@
 )
 
 try:
-    from pymongocrypt.auto_encrypter import AutoEncrypter
-    from pymongocrypt.errors import MongoCryptError
-    from pymongocrypt.explicit_encrypter import ExplicitEncrypter
-    from pymongocrypt.mongocrypt import MongoCryptOptions
-    from pymongocrypt.state_machine import MongoCryptCallback
+    from pymongocrypt.auto_encrypter import AutoEncrypter  # type:ignore[import]
```

Comment:
Why change from having this in the mypy setting to having it in the code?

Reply:
To me it is clearer to see it at the import, we look there more often than at config.

Reply:
Fair.